packages/benchmark-wasm/benchmarks/hover.bench.ts

```diff
@@ -21,26 +21,33 @@ h1.foo {
 h1 > span {
   color: linear-gradient(to right, red, #fff);
 }
-`
+`;
 
-  const textDocument = TextDocument.create(
-    "file:///test.css",
-    "css",
-    0,
-    content
-  )
+	const textDocument = TextDocument.create(
+		"file:///test.css",
+		"css",
+		0,
+		content
+	);
+	const ls = new LanguageService({
+		include_base_css_custom_data: true,
+	});
+	await ls.upsert_document(textDocument);
 
-  bench("CSSLSRS(WASM) - Hover", async () => {
-    await get_hover(textDocument, { line: 4, character: 3 })
-  })
-  if (!process.env.CODSPEED) {
-    bench("vscode-css-languageservice - Hover", () => {
-      const stylesheet = vscodeLanguageService.parseStylesheet(textDocument)
-      vscodeLanguageService.doHover(
-        textDocument,
-        { line: 4, character: 3 },
-        stylesheet
-      )
-    })
-  }
-})
+	bench("CSSLSRS(WASM) - Hover", async () => {
+		await ls.get_hover(textDocument.uri, {
+			line: 14,
+			character: 3,
```

Comment:
The ranges pointed to an empty line here, causing this to be a benchmark of who can return undefined or null the fastest.